pages/glossary.md

```diff
@@ -61,6 +77,14 @@ A Safe owner is one of the accounts that control a given Safe. Only owners can m
 See also:
 - [OwnerManager.sol](https://github.com/safe-global/safe-smart-account/blob/main/contracts/base/OwnerManager.sol) on github.com
 
+## Paymaster
+
+Paymasters are smart contracts that allow an entity to pay for the gas fees of other users. This feature abstracts away the concept of gas fees by subsidizing them for users, allowing them to pay with ERC-20 tokens, and many other use cases.
```

Comment:
Can we specify what entity means?

Reply:
Entity is a company, project, person, or anybody interacting

Reply:
Thoughts on adding this in parathesis or potentially defining entity as a term?

Reply:
Maybe we can replace entity with "account"